Client/Frontend/Fakespot/Views/FakespotMessageCardView.swift

```diff
@@ -73,6 +73,24 @@ struct FakespotMessageCardViewModel {
         }
     }
 
+    func sendTelemetryOnAppear() {
+        switch self.type {
+        case .confirmation:
+            return
+        case .warning:
+            return
+        case .info:
+            return
+        case .infoLoading:
+            return
+        case .error:
+            return
+        case .infoTransparent:
+            TelemetryWrapper.recordEvent(category: .action, method: .view, object: .shoppingSurfaceStaleAnalysisShown)
```

Comment:
The infoTransparent card could be used for other things. So I don't think this is a good place for this telemetry. I'd suggest to send it in the `FakespotViewController` when the card is created instead.